Src/WitsmlExplorer.Frontend/contexts/navigationStateReducer.tsx

```diff
@@ -371,6 +385,29 @@ const updateWellboreMessage = (state: NavigationState, { payload }: UpdateWellbo
     selectedMessage
   };
 };
+const updateWellboreRisks = (state: NavigationState, { payload }: UpdateWellboreRisksAction) => {
```

Comment:
These are unrelated to the issue and not used, please remove updateWellboreRisks and updateWellboreRisk